pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/Utils.java

```diff
@@ -206,4 +207,14 @@ public static String getFullyQualifiedInstanceId(String tenant, String namespace
                                                      String functionName, int instanceId) {
         return String.format("%s/%s/%s:%d", tenant, namespace, functionName, instanceId);
     }
+
+    public static boolean deleteDirectory(File directoryToBeDeleted) {
```

Comment:
I think deleteRecursively has been available in at some libraries already.

e.g. guava - https://google.github.io/guava/releases/21.0/api/docs/com/google/common/io/MoreFiles.html#deleteRecursively-java.nio.file.Path-com.google.common.io.RecursiveDeleteOption...-

commons-io : https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#deleteDirectory(java.io.File)